### PR TITLE
Fix Hotjar & Hubspot Integrations

### PR DESCRIPTION
**Pull request by kirananto**

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -28,6 +28,25 @@ module.exports = {
       }
     ]
   ],
+  head: [
+    [
+      "script",
+      { src: "//js.hs-scripts.com/5868902.js", async: true, defer: true }
```
Comment (bmehendale) [inline docs/.vuepress/config.js]: should this be at the end of the file, before the closing </html> tag?

Comment (bmehendale) [reply docs/.vuepress/config.js]: also, i looked at view-source:https://docs-develop.netlify.com/home/espercli.html and didn't see this .js file.. should i wait and refresh?

Comment (kirananto) [reply docs/.vuepress/config.js]: Nope since this is asyc true, it'll only load in an async fashion.. 👍

Comment (kirananto) [reply docs/.vuepress/config.js]: https://www.growingwiththeweb.com/2014/02/async-vs-defer-attributes.html

^^ This shows how the performance is not altered by the script by using async and defer true...

Comment (kirananto) [reply docs/.vuepress/config.js]: image

https://deploy-preview-113--docs-develop.netlify.com/ --> Check here deploy preview... it's not yet in develop

**Pull request merged**